addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -129,6 +129,13 @@ export class ColorPlugin extends Plugin {
             applyColorResetPreview: this.applyColorResetPreview.bind(this),
             colorPrefix: mode === "color" ? "text-" : "bg-",
             onClose: () => this.dependencies.selection.focusEditable(),
+            getTargetedElements: () => {
+                const nodes = this.dependencies.selection.getTargetedNodes().filter(isTextNode);
+                if (nodes.length === 0) {
+                    return;
+                }
+                return nodes.map((node) => closestElement(node));
```

Comment:
What's the point of this? Why not return `nodes.map((node) => closestElement(node));` and it will be an empty list anyway? If `getTargetedElements` always returns a list, even if an empty one, it should simplify the code where you call it because you know you will always get a list so you don't have to check for falsy values.